[2022-03-21 22:03:16,729] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: EL_dag.extract_task scheduled__2012-02-02T01:00:00+00:00 [queued]>
[2022-03-21 22:03:16,735] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: EL_dag.extract_task scheduled__2012-02-02T01:00:00+00:00 [queued]>
[2022-03-21 22:03:16,735] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-21 22:03:16,735] {taskinstance.py:1244} INFO - Starting attempt 6 of 7
[2022-03-21 22:03:16,736] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-21 22:03:16,744] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): extract_task> on 2012-02-02 01:00:00+00:00
[2022-03-21 22:03:16,748] {standard_task_runner.py:52} INFO - Started process 3673 to run task
[2022-03-21 22:03:16,751] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'EL_dag', 'extract_task', 'scheduled__2012-02-02T01:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/extract_load_dags.py', '--cfg-path', '/tmp/tmpxl3lbzii', '--error-file', '/tmp/tmp52k5opof']
[2022-03-21 22:03:16,752] {standard_task_runner.py:77} INFO - Job 49: Subtask extract_task
[2022-03-21 22:03:16,881] {logging_mixin.py:109} INFO - Running <TaskInstance: EL_dag.extract_task scheduled__2012-02-02T01:00:00+00:00 [running]> on host 7fae4c5ec6da
[2022-03-21 22:03:16,908] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.9/site-packages/airflow/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-03-21 22:03:16,933] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=EL_dag
AIRFLOW_CTX_TASK_ID=extract_task
AIRFLOW_CTX_EXECUTION_DATE=2012-02-02T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2012-02-02T01:00:00+00:00
[2022-03-21 22:03:16,934] {logging_mixin.py:109} INFO - 02 2012
[2022-03-21 22:03:19,896] {logging_mixin.py:109} INFO - the response is  200
[2022-03-21 22:03:19,896] {logging_mixin.py:109} INFO - here.........
[2022-03-21 22:03:19,933] {logging_mixin.py:109} INFO -   invoice_line_no                     date  ... sale_liters sale_gallons
0    S04013700033  2012-02-09T00:00:00.000  ...        1.00         0.26
1    S04325000004  2012-02-29T00:00:00.000  ...       10.50         2.77
2    S03914700072  2012-02-06T00:00:00.000  ...        1.50         0.40
3    S04276100043  2012-02-27T00:00:00.000  ...       12.00         3.17
4    S03874200049  2012-02-01T00:00:00.000  ...        3.50         0.92

[5 rows x 24 columns]
[2022-03-21 22:03:19,950] {logging_mixin.py:109} INFO - Data saved..........
[2022-03-21 22:03:19,952] {python.py:175} INFO - Done. Returned value was: None
[2022-03-21 22:03:19,963] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=EL_dag, task_id=extract_task, execution_date=20120202T010000, start_date=20220321T220316, end_date=20220321T220319
[2022-03-21 22:03:20,017] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-21 22:03:20,134] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
